sh/install.sh

```diff
@@ -206,5 +264,5 @@ $WAVE_PATH $cli_args
 
 # Python was not installed before running this script, so remove it.
 if [ $PYTHON_INSTALLED == false ]; then
-    remove_python
+    remove_python $PIP_PATH
```

Comment:
I see you've initialized the `PYTHON3_INSTALLED` above but keep using `PYTHON_INSTALLED` here

Reply:
Removed the unused variable **PYTHON3_INSTALLED**